yb-voyager/cmd/exportData.go

```diff
@@ -595,10 +604,10 @@ func renameDbzmExportedDataFiles() error {
 		}
 
 		//rename table schema file as well
-		oldTableSchemaFilePath := filepath.Join(exportDir, "data", "schemas", strings.Replace(status.Tables[i].FileName, "_data.sql", "_schema.json", 1))
-		newTableSchemaFilePath := filepath.Join(exportDir, "data", "schemas", tableName+"_schema.json")
+		oldTableSchemaFilePath := filepath.Join(exportDir, "data", "schemas", SOURCE_DB, strings.Replace(status.Tables[i].FileName, "_data.sql", "_schema.json", 1))
```

Comment:
I hope we're preserving the case sensitivity of these file-names even while exporting schemas from TARGET.

Reply:
Nope, that's yet to be handled - which is what @sanyamsinghal is going to work on.